### PR TITLE
[EN] timelines: English the article

### PR DESCRIPTION
**Pull request by MegaApplePi**
- English the article

### CODE REVIEW

File: wiki/Beatmap_Editor/Timelines/en.md [hunk 1]
```diff
@@ -1,137 +1,141 @@
...
-| Colour | Description |
-| ------ | ----------- |
-| Bright White Long	| Current test section |
-| Yellow Long	| Preview point. |
-| Yellow up	| Start of Drain time. |
-| Green up | Inherited Points. (See [Timing Setup](/wiki/Beatmap_Editor/Timing)) |
-| Red up | Timing Points. (See Timing Setup) |
-| Blue Down |	Bookmark |
+In the centre, it shows the timeline with markings and the compulsory music player buttons.
+The "Test" button will save your beatmap then will start play-testing it starting from the current timestamp.
```
Comment (EpicSupermind) [inline wiki/Beatmap_Editor/Timelines/en.md]: `Test` in code

File: wiki/Beatmap_Editor/Timelines/en.md [hunk 2]
```diff
@@ -1,137 +1,141 @@
...
-*Location: Top @ [Compose](/wiki/Beatmap_Editor/Compose) tab in any modes other than osu!mania only.*
+### osu!, osu!taiko, and osu!catch
 
-![Hit Objects Timeline](/wiki/shared/BE_NTL.jpg "Hit Objects Timeline")
+![Hit Objects Timeline in osu!, osu!taiko, and osu!catch](/wiki/shared/BE_NTL.jpg "This shows the timeline for hit objects with respect to the beat snap divisor and timestamp.")
 
-**Shows the timeline for hitobjects with respect to [Beat Snap Divisor](/wiki/Beatmap_Editor/Beat_Snap_Divisor) and timestamp.**
-
-#### Hit Object Timeline
+In compose mode, this timeline is underneath the [Compose](/wiki/Compose) tab for any game mode except for [osu!mania](wiki/osu!mania).
```
Comment (EpicSupermind) [inline wiki/Beatmap_Editor/Timelines/en.md]: `[`Compose`](/wiki/Compose) tab`?

File: wiki/Beatmap_Editor/Timelines/en.md [hunk 3]
```diff
@@ -1,137 +1,141 @@
...
 ![osu!mania playfield](/wiki/shared/BEM_PF.jpg "osu!mania playfield")
 
-#### Playfield Boxes
+In compose mode, this timeline is in the centre of the playfield for osu!mania mode.
 
-| Name | Description |
-| ---- | ----------- |
-| Left Box | Notes intensity (Timeline). |
-| Centre | Actual Playfield ([Beat Snap Divisor](/wiki/Beatmap_Editor/Beat_Snap_Divisor)). |
+The box on the left is a horizontal bar char that displays the note intensity.
```
Comment (EpicSupermind) [inline wiki/Beatmap_Editor/Timelines/en.md]: bar char? I might be wrong here, but what's a char?

File: wiki/Beatmap_Editor/Timelines/en.md [hunk 4]
```diff
@@ -1,137 +1,141 @@
...
+| White/Pink/Yellow | Unselected note colours |
 
-*Location: Top @ [Design](/wiki/Beatmap_Editor/Design) tab*
+## Design
 
-**Shows the timeline for transformations of the selected object.**
+![Storyboard Timeline](/wiki/shared/SE_STM.jpg "This shows the timeline for transformations of the selected sprite.")
 
-![Storyboard Timeline](/wiki/shared/SE_STM.jpg "Storyboard Timeline")
+The design timeline is located underneath the [Design](/wiki/design) tab.
```
Comment (EpicSupermind) [inline wiki/Beatmap_Editor/Timelines/en.md]: `[`Design`](/wiki/design) tab.` ? you don't have to accept this one or the compose one, idk

File: wiki/Beatmap_Editor/Timelines/en.md [hunk 5]
```diff
@@ -1,137 +1,141 @@
...
 
-*Location: [Compose](/wiki/Beatmap_Editor/Compose), [Design](/wiki/Beatmap_Editor/Design), [Timing](/wiki/Beatmap_Editor/Timing) (Bottom)*
+The song player is visible in any part of the beatmap editor.
 
-At **left**, it shows **timestamp in milliseconds (ms)** and **song duration %**. The "%" may be switched to "intro/outro" when there is intro/outro storyboarding.
+### Left
 
-At **centre**, it shows the **timeline with markings and the compulsory music player buttons.** For "Test" button, your beatmap will be saved to test the beatmap starting on current timestamp.
+On the left, there is the timestamp in milliseconds and the song duration percentage.
+The percentage may display "intro" or "outro" if there is storyboarding before or after the song.
```
Comment (kj415j45) [inline wiki/Beatmap_Editor/Timelines/en.md]: code `` `intro` `` and `` `outro` `` (?)

File: wiki/Beatmap_Editor/Timelines/en.md (hunk 3 above)
Comment (MegaApplePi) [inline wiki/Beatmap_Editor/Timelines/en.md]: that is a typo ... bar char`t`

File: wiki/Beatmap_Editor/Timelines/en.md [hunk 6]
```diff
@@ -1,137 +1,135 @@
-Timelines
-=========
+# Timelines
 
-This will list all the possible timelines you will encounter in Beatmap Editor along with their respective location.
+In the [beatmap editor](/wiki/beatmap_editor), there are three different kinds of timelines that a mapper may face.
```
Comment (EpicSupermind) [inline wiki/Beatmap_Editor/Timelines/en.md]: `there a three different timelines`?

File: wiki/Beatmap_Editor/Timelines/en.md [hunk 7]
```diff
@@ -1,137 +1,135 @@
...
 
 ### Timeline
 
 | Name | Description |
 | ---- | ----------- |
-| `+`/`-` buttons at left	| Increase/Decrease timeline zoom |
-| `Up`/`Down` arrow buttons left-below | Scroll up/down the transformation timeline (to see Move/Colour timeline). |
-| Centre | Transformation Timeline for the selected SB object. |
+| `+`/`-` buttons on left | Increase/Decrease timeline zoom. |
+| `Up`/`Down` arrows on bottom-left | Scroll up/down the transformation timeline (this is to reveal Colour or Movement on the timeline). |
```
Comment (EpicSupermind) [inline wiki/Beatmap_Editor/Timelines/en.md]: `(this is to reveal colour or movement on the timeline)`

File: wiki/Beatmap_Editor/Timelines/en.md [hunk 8]
```diff
@@ -1,137 +1,135 @@
...
+If there is a transformation, that transformation is lighted with it's colour.
+In addition to this, it will have two separated half-lines that determines the duration.
+Full white line determines the transformation switching points (e.g. from "going up" to "going down").
 
-*Location: Top @ [Timing](/wiki/Beatmap_Editor/Timing) tab*
+## Timing
 
 ![Timing Timeline](img/TT.jpg "Timing Timeline")
 
+The timing timeline is located underneath the [Timing](/wiki/Timing) tab.
```
Comment (EpicSupermind) [inline wiki/Beatmap_Editor/Timelines/en.md]: `` [`Timing`](/wiki/Timing) tab.``

File: wiki/Beatmap_Editor/Timelines/en.md (hunk 6 above)
Comment (TPGPL) [inline wiki/Beatmap_Editor/Timelines/en.md]: idk, capitalize beatmap editor?

Comment (TPGPL) [inline wiki/Beatmap_Editor/Timelines/en.md]: face -> encounter

File: wiki/Beatmap_Editor/Timelines/en.md [hunk 9]
```diff
@@ -1,137 +1,135 @@
...
-This will list all the possible timelines you will encounter in Beatmap Editor along with their respective location.
+In the [beatmap editor](/wiki/beatmap_editor), there are three different kinds of timelines that a mapper may face.
+This article will explain how each of them function.
 
-Song Player
-------------
+## Shortcuts
 
-![Song's Timeline](/wiki/shared/BE_STL.jpg "Song's Timeline")
+_For a list of keyboard shortcuts for the timeline, see [shortcut key reference](/wiki/skr#general)._
```
Comment (TPGPL) [inline wiki/Beatmap_Editor/Timelines/en.md]: not /wiki/skr/#general?

File: wiki/Beatmap_Editor/Timelines/en.md [hunk 10]
```diff
@@ -1,137 +1,135 @@
...
-*Location: Top @ [Compose](/wiki/Beatmap_Editor/Compose) tab in any modes other than osu!mania only.*
+### osu!, osu!taiko, and osu!catch
 
-![Hit Objects Timeline](/wiki/shared/BE_NTL.jpg "Hit Objects Timeline")
+![Hit Objects Timeline in osu!, osu!taiko, and osu!catch](/wiki/shared/BE_NTL.jpg "This shows the timeline for hit objects with respect to the beat snap divisor and timestamp.")
 
-**Shows the timeline for hitobjects with respect to [Beat Snap Divisor](/wiki/Beatmap_Editor/Beat_Snap_Divisor) and timestamp.**
-
-#### Hit Object Timeline
+In [compose](/wiki/Compose) mode, this timeline is underneath the `Compose` tab for any game mode except for [osu!mania](wiki/osu!mania).
```
Comment (TPGPL) [inline wiki/Beatmap_Editor/Timelines/en.md]: capitalize compose

File: wiki/Beatmap_Editor/Timelines/en.md (hunk 9 above)
Comment (MegaApplePi) [inline wiki/Beatmap_Editor/Timelines/en.md]: doesn't really matter but will change

File: wiki/Beatmap_Editor/Timelines/en.md (hunk 10 above)
Comment (MegaApplePi) [inline wiki/Beatmap_Editor/Timelines/en.md]: The first compose is talking about compose mode, not the article itself.
I will lowercase the c in compose (link) tho.

**Pull request merged**